layersvt/multi.cpp

```diff
@@ -26,11 +26,41 @@
 #include "vk_loader_platform.h"
 #include "vulkan/vk_layer.h"
 #include "vk_layer_table.h"
+#include "vk_layer_extension_utils.h"
```

Comment:
Please copy the functions util_GetExtensionProperties  and util_GetLayerProperties directly in this file rather than making this layer dependent upon the layer utils library.  This is an example layer intended to be copied by layer authors to their own repository. It is best if it is standalone  and doesn't pull in a large set of utilities most of which are not necessarily needed by the layer author.